cli/agent.go

```diff
@@ -323,8 +323,9 @@ func (r *RootCmd) workspaceAgent() *clibase.Cmd {
 			Value:       clibase.BoolOf(&noReap),
 		},
 		{
-			Flag:        "ssh-max-timeout",
-			Default:     "0",
+			Flag: "ssh-max-timeout",
+			// tcpip.KeepaliveIdleOption = 72h (forwardTCPSockOpts() in tailnet/conn.go)
+			Default:     fmt.Sprintf("%v", 72*time.Hour-time.Minute),
```

Comment:
Two thoughts:
1. How about we set this to an even 72h, but do the subtraction elsewhere? I think the even number will be easier for users to understand
    - Or, we could bump the tailnet conn keepalive slightly to account for this
3. Perhaps the description should mention this is the maximum sensible value for this field?

Reply:
1. I set it to 72h, and changed `tcpip.KeepaliveIdleOption` to 72h + 1min as it seems straightforward to me. I guess that it doesn't make any difference that we elongate the TCP timeout by 1 minute.
2. Done (range: 60s - 72h).